src/test/java/teammates/test/cases/common/StringHelperTest.java

```diff
@@ -248,6 +248,20 @@ public void testTruncateHead() {
         assertEquals("123456789", StringHelper.truncateHead("123456789", 10));
         assertEquals("567890", StringHelper.truncateHead("1234567890", 6));
     }
+    
+    @Test
+    public void testRemoveEnclosingSquareBrackets() {
+        assertEquals("test1, test2", StringHelper.removeEnclosingSquareBrackets("[test1, test2]"));
+        assertEquals("test1,test2", StringHelper.removeEnclosingSquareBrackets("  [test1,test2] "));
+        assertEquals("[ \"test\" ]", StringHelper.removeEnclosingSquareBrackets("[ [ \"test\" ]]"));
+        assertEquals("test1,[], ][test2]", 
+            StringHelper.removeEnclosingSquareBrackets(" [test1,[], ][test2]]   "));
+        assertEquals("test", StringHelper.removeEnclosingSquareBrackets("test"));
+        assertEquals("  [test  ", StringHelper.removeEnclosingSquareBrackets("  [test  "));
+        assertEquals("  (test]  ", StringHelper.removeEnclosingSquareBrackets("  (test]  "));
+        assertEquals("", StringHelper.removeEnclosingSquareBrackets(""));
+        assertEquals(null, StringHelper.removeEnclosingSquareBrackets(null));
```

Comment:
You can document what each test case is, either as a comment, or using the 3-parameter version of the `assertEquals` that takes an explanatory message as a parameter.